docs/setup/plugins/discourse.md

```diff
@@ -1,5 +1,121 @@
 ---
 title: Discourse
+description: Setting up the Discourse plugin.
 ---
 
-TODO
+## Status and Caveats
+
+This plugin is currently in Beta. It assigns Cred scores that are reasonable and
+robust for a [trust level] 3 community.
+
+Currently there are two general approaches to minting Cred you can take with the
+Discourse plugin. Activity-minted and like-minted Cred (or a hybrid of the two).
+Activity-minted Cred means Cred would be minted for each new topic or post
+created. While like-minted Cred mints new Cred for each like given.
+
+The plugin defaults to using like-minted Cred and for most communities this will
+be the better approach. Unlike activity-minted Cred this allows people to
+validate contributions. People soon learn their likes work as "this is valuable
+and I'd like to see more of this" signals, which incentivizes creating
+high-quality posts.
+
+The caveat is that it creates some "popularity contest" dynamics. Where memes
+and/or heavily promoted posts might receive more likes than makes sense for the
+relative value they've added. Something which would be easy to game, making it
+less suitable for lower trust levels for the time being.
+
+Another thing to keep in mind, is that private forums are not included for Cred
+calculation.
```

Comment:
This could be confusing because it sounds like the whole forum (i.e. the whole discourse) is private. I suggest saying that only publicly viewable posts are included in cred calculation.

Reply:
Good point. I'm thinking of other forum software / conventions where sub-forums are also considered forums, indeed being ambiguous. Discourse calls them categories. But more private data (such as private messages) are not included, so switching to "only public" rather than "no private" is probably easier to explain :]

Reply:
Updated this, and expanded on the motivation while at it.